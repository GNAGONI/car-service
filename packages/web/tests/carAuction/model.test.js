import React from 'react'
import renderer from 'react-test-renderer'
import Model from '../../components/carAuction/model'

it('renders correctly', () => {
  const tree = renderer.create(<Model />).toJSON()
  expect(tree).toMatchSnapshot()
})
