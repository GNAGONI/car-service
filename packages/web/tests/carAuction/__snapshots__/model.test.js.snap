// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="model-container container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <section
        className="content extend section-model bg-before-info"
      />
    </div>
  </div>
</div>
`;
